### PR TITLE
forgot to do pull request for read-pair

### PR DESCRIPTION
**Pull request by sgosline**

### CODE REVIEW

**Comment by karawoo** [comment]
@sgosline would it be possible for you to revert e850bc73a16c7813716ef58149b2c24031effa0d so we can review the whole thing together? Annotation keys cannot contain `-` characters.

**Comment by sgosline** [comment]
ok, i think i fixed it and moved to camel case - can you review @karawoo ?

**Comment by karawoo** [comment]
I think you need to add the full entry on this branch since it's no longer present on develop.

**Comment by sgosline** [comment]
Thanks @karawoo - @jaeddy or @kdaily can you review? or @karawoo ?

File: synapseAnnotations/data/ngs.json [hunk 1]
```diff
@@ -17,6 +17,24 @@
       }
     ]
   },
+  {
+      "name": "readPair",
+      "description": "The read of origin",
+      "columnType": "STRING",
```
Comment (kdaily) [inline synapseAnnotations/data/ngs.json]: why is it a string?

File: synapseAnnotations/data/ngs.json [hunk 2]
```diff
@@ -17,6 +17,24 @@
...
+      "maximumSize": 5,
```
Comment (kdaily) [inline synapseAnnotations/data/ngs.json]: if it's a string (which maybe it doesn't need to be) then the maximum size should be 1?

Comment (sgosline) [reply synapseAnnotations/data/ngs.json]: Because I wanted to be flexible for other inputs, like 'unknown'? I dunno...

Comment (sgosline) [reply synapseAnnotations/data/ngs.json]: or 'R1' as kara said, but i'm happy to stick to integer.

**Comment by allaway** [comment]
Hey, isn't there already a key for this (`readPairOrientiation`)?

**Comment by allaway** [comment]
er, maybe `readPairOrigin`?

**Comment by sgosline** [comment]
I think that is for forward/reverse strandedness, not for mates of a read. But maybe i have my term confused?

**Comment by allaway** [comment]
Right, whoops. I think I just confused myself. These would be two different
annotations unless the library prep is a strand-specific one, i suppose.

File: synapseAnnotations/data/ngs.json (hunk 1 above)
Comment (sgosline) [inline synapseAnnotations/data/ngs.json]:
```suggestion
      "columnType": "INTEGER",
```

File: synapseAnnotations/data/ngs.json [hunk 3]
```diff
@@ -17,6 +17,24 @@
       }
     ]
   },
+  {
+      "name": "readPair",
+      "description": "The read of origin",
+      "columnType": "INTEGER",
+      "maximumSize": 5,
```
Comment (sgosline) [inline synapseAnnotations/data/ngs.json]:
```suggestion
```

File: synapseAnnotations/data/ngs.json [hunk 4]
```diff
@@ -17,6 +17,23 @@
       }
     ]
   },
+  {
+      "name": "readPair",
+      "description": "The read of origin",
+      "columnType": "INTEGER",
+      "enumValues": [
+        {
+          "value": "1",
```
Comment (sgosline) [inline synapseAnnotations/data/ngs.json]:
```suggestion
          "value": 1,
```

File: synapseAnnotations/data/ngs.json [hunk 5]
```diff
@@ -17,6 +17,23 @@
...
+      "description": "The read of origin",
+      "columnType": "INTEGER",
+      "enumValues": [
+        {
+          "value": 1,
+          "description": "One of a pair of sequencing reads in which the two members of the pair are related by originating at either end of a clone insert.",
+          "source": "http://purl.obolibrary.org/obo/SO_0000007"
+        },
+        {
+          "value": "2",
```
Comment (sgosline) [inline synapseAnnotations/data/ngs.json]:
```suggestion
          "value": 2,
```

**Pull request merged**